styles.css

```diff
@@ -584,6 +589,45 @@ nav li a img {
   color: white;
 }
 
+.mobile-menu {
+  min-height: 718px;
+  width: 0;
+  display: none;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  position: fixed;
+  top: 0;
+  background: #0e23dda9;
+  backdrop-filter: blur(5px);
+  transition: 0.6s;
+}
+
+.mobile-menu ul {
+  list-style-type: none;
+  margin: 10% 5%;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+#cross {
+  margin: 3% 3%;
+}
```

Comment:
- [ ] **Good job on implementing the hamburger button 👍**
   However, It would be better also to follow the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=fkkWgg7N5bDssBAb-0) by matching your close button's color to that of the Figma design 😀😀

   This will also make it more visible for the best user experience 👍 👍  

    **Your design** | **Figma design**
   | --- | --- |
   ![image](https://user-images.githubusercontent.com/102757126/217042552-57019dd7-ffb2-4548-95cf-2602a0b3907e.png) | ![image](https://user-images.githubusercontent.com/102757126/217042651-1cb0257d-818e-4b88-a259-d50caa0fba5e.png)

   _Certainly! Following a consistent design language, such as the Figma template, helps ensure a cohesive and professional look for your project. Matching the color of the close button to the Figma design is a great way to maintain consistency. ✨✨_

Reply:
Cross Image changed and taken from Figma Desgin. Changes made